lib/rubocop/cop/rspec/mixin/top_level_group.rb

```diff
@@ -4,6 +4,7 @@ module RuboCop
   module Cop
     module RSpec
       # Helper methods for top level example group cops
+      # @api private
       module TopLevelGroup
```

Comment:
I can argue that this one is not private - it's just not documented. We mention it [here](https://github.com/rubocop/rubocop-rspec/blob/808ce138ca1a69e13fb33f0eff097c5a614b1b98/docs/modules/ROOT/pages/upgrade_to_version_2.adoc#replace-topleveldescribe) to be used on custom RSpec cops.

I can't tell which other mixins/helpers are really private, or can be used by other projects with no fear of them being removed in a minor release. But I'm on the fence of marking them all as private.

Reply:
Oh, thank you. I had overlooked that document. And it is true that these mixins may not as private.